test/WormholeL2GovernorMetadata.t.sol

```diff
@@ -61,7 +65,8 @@ contract L2GovernorMetadataTest is TestConstants {
 
 contract Constructor is L2GovernorMetadataTest {
   function testFuzz_CorrectlySetsAllArgs(address wormholeCore) public {
-    new WormholeL2GovernorMetadata(wormholeCore, msg.sender); // nothing to assert as there are no
+    new WormholeL2GovernorMetadata(wormholeCore, msg.sender, address(0x1b)); // nothing to assert as
+      // there are no
       // constructor args set
```

Comment:
Nit: Maybe merge these comments onto one line

Reply:
Done in 165e503186e3748ec195858fcb3972d1021c0610